org.eclipse.swtchart.extensions/src/org/eclipse/swtchart/extensions/Messages_fr.properties

```diff
@@ -0,0 +1,9 @@
+# ----------------------------------------------------------------------
+# Properties for org.eclipse.swtchart.extensions
+#
+# Entries are of the form CONSTANT=string
+# ----------------------------------------------------------------------
+# ----------------------------------------------------------------------
+# Properties used by Contextual menu items
+# ----------------------------------------------------------------------
+RESET_CHART_LABEL = R�initiliser le Zoom (1:1)
```

Comment:
are these really the only labels/string used in SWTChart?

Reply:
As far as I understood Frank, he asked if this is the correct way to internationalize the bundle. If ok, he will proceed with translations.

Reply:
I see. org.eclipse.osgi.util.NLS is an eclipse extension, so if you are fine with binding to Eclipse-API thats the way to go, otherwise I would suggest to use plain java internationalization even though its a little bit more verbose

Reply:
+1
If possible, we should avoid dependencies as best as possible, so that SWTChart can be used in many environments.

Reply:
+1

Reply:
I've proceed internationalization for these bundles (without any dependencies except on java.util) :
- swtchart
- swtchart.export
- swtchart.extensions

Please let me know if you see any problem and if you think these bundles should also be internationalized :
- swtchart.examples
- swtchart.extensions.examples